modules/platforms/dotnet/Apache.Ignite.Core.Tests/ApiParity/StreamerParityTest.cs

```diff
@@ -29,7 +29,8 @@ public class StreamerParityTest
         /** Members that are not needed on .NET side. */
         private static readonly string[] UnneededMembers =
         {
-            "deployClass"
+            "deployClass",
+            "perBatchBufferSize"
```

Comment:
Why `perBatchBufferSize` added to `UnneededMembers`?

Reply:
this method should be ignored in .NET tests